frontend/tests/integration/components/user-component/preview-test.js

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
@gabru-md can we remove this? Since the entire file is commented out.

Reply:
this is a test file. We will work on it in the coming days. Therefore it has been added. @djmgit :sweat_smile: